web/src/constants.ts

```diff
@@ -161,3 +161,14 @@ export const MAXIMUM_TECHNICAL_LEADS = 2;
 
 export const PRODUCT_OWNER_SUBTITLE = 'This is typically the business owner of the application;';
 export const TECHNICAL_LEAD_SUBTITLE = 'This is typically the DevOps specialist;';
+
+export const NAMESPACE_URLS = {
+  PROD_GOLD: process.env.REACT_APP_PRODUCTION_NAMESPACE_GOLD_URL,
```

Comment:
This is what I am referring in our DEV channel. React app don't really pick up env variable in openshift containerize environment.